sdk/ai/azure-ai-projects/README.md

```diff
@@ -1,53 +1,188 @@
+```python
+import os
+from azure.ai.projects import AIProjectClient
+from azure.identity import DefaultAzureCredential
 
-Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
-`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+```
 
-Use the returned token credential to authenticate the client:
+To construct an asynchronous client, Install the additional package [aiohttp](https://pypi.org/project/aiohttp/):
 
-```python
->>> from azure.ai.projects import AIProjectClient
->>> from azure.identity import DefaultAzureCredential
->>> client = AIProjectClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+```bash
+pip install aiohttp
 ```
 
-## Key concepts
+and update the code above to import `asyncio`, and import `AIProjectClient` from the `azure.ai.projects.aio` namespace:
 
-TODO
+```python
+import os
+import asyncio
+from azure.ai.projects.aio import AIProjectClient
+from azure.core.credentials import AzureKeyCredential
+
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+```
 
 ## Examples
 
+### Enumerate connections
+
+You Azure AI Studio project has a "Management center". When you enter it, you will see a tab named "Connected resources" under your project. The `.connections` operations on the client allow you to enumerate the connections and get connection properties. Connection properties include the resource URL and authentication credentials, among other things.
+
+#### Get properties of all connections
+
+To list the properties of all the connections in the Azure AI Studio project:
+
+```python
+connections = project_client.connections.list()
+for connection in connections:
+    print(connection)
+```
+
+#### Get properties of all connections of a particular type
+
+To list the properties of connections of a certain type (here Azure OpenAI):
+
+```python
+connections = project_client.connections.list(
+    connection_type=ConnectionType.AZURE_OPEN_AI,
+)
+for connection in connections:
+    print(connection)
+
+```
+
+#### Get properties of a default connection
+
+To get the properties of the default connection of a certain type (here Azure OpenAI),
+with its authentication credentials:
+
+```python
+connection = project_client.connections.get_default(
+    connection_type=ConnectionType.AZURE_OPEN_AI,
+    include_credentials=True,  # Optional. Defaults to "False"
+)
+print(connection)
+```
+
+If the call was made with `include_credentials=True`, depending on the value of `connection.authentication_type`, either `connection.key` or `connection.token_credential`
+will be populated. Otherwise both will be `None`.
+
+#### Get proprties of a connection by its connection name
```

Comment:
small typo proprties

Reply:
Thanks! not sure what happened to my speller. Not working. I fixed this and scanned for other spelling mistakes.